stop.sh

```diff
@@ -1,8 +1,6 @@
 #!/bin/bash
 
 echo "Shutting down Akeneo service"
-source .env
-export SOURCE_PATH="${WORKING_DIR}"
 docker-compose down
```

Comment:
I've updated the README to indicate it's just a wrapper for now. But in future this might chance, majorly because `docker-machine start` can not always guarantee same IPs across different sessions, so in future, the script may be improved to alter the hosts' `/etc/hosts`. But, I hope all `su` operations can be handled outside this setup. So this is still up for debate for now as to when this implementation will be. But using the start and stop scripts is recommended.